config/stack/manifests/resources/yugabyteclusterclass.resource.yaml

```diff
@@ -6,4 +6,4 @@ overviewShort: "A YugabyteClusterClass is a resource class. It defines the desir
 overview: |
   A YugabyteClusterClass is a resource class. It defines the desired spec of resource claims that use it to dynamically provision a managed resource.
 readme: |
-  Full documentation can be found in the Crossplane [API Reference docs](https://crossplane.io/docs/master/api/crossplane/stack-rook/storage-rook-crossplane-io-v1alpha1.html#YugabyteClusterClass).
```

Comment:
were the API docs already updated, or will those need to be completely regenerated for all repos in one big PR?

Reply:
they will need to be regenerated after all of the PR’s like this one are merged on each provider. I plan on those following this already open PR: https://github.com/crossplane/crossplane/pull/1322